zeebe/exporters/elasticsearch-exporter/src/main/resources/zeebe-record-process-template.json

```diff
@@ -45,6 +45,9 @@
             },
             "deploymentKey": {
               "type": "long"
+            },
+            "versionTag": {
+              "type": "keyword"
```

Comment:
FYI: I was a bit unsure if I should use type `keyword` or `text`, but from https://www.elastic.co/guide/en/elasticsearch/reference/current/keyword.html it sounded like `keyword` would be more appropriate:

> used for structured content such as IDs, email addresses, hostnames, status codes, zip codes, or **tags**.